### ISSUE TITLE
 adaptive voxel size and  degeneracy detection factor not open sourced, right? 

### ISSUE THREAD

**Issue by narutojxl** [jisehua/Degenerate-Detection#4]
Dear authors @jisehua ,
The open sourced repo doesn't  include the implementation of paper III-A "Adaptive Adjustment of Voxel Size" and III-B " Degeneracy Detection Factor" part,  in voxel_grid_covariance_omp_impl.hpp and ndt_omp_impl.hpp,  compared with  https://github.com/koide3/ndt_omp, right?

**Comment by jisehua** [comment]
Yes, thanks for your interest in our work, we'll get the rest of the code sorted and uploaded as soon as possible.
